### PR TITLE
fix variable height to elevationAngle

### LINKED ISSUE #100: Adapt to changed `PvInputModel`
As of PowerSystemDataModel the parameter `height` changed to `elevationAngle`

### PR DESCRIPTION
**Pull request by danielfeismann**
Resolves #100 

Also fixed: 
- Resolved an issue with depricated time series
- Change weather sceme to COSMO

### CODE REVIEW

**Comment by sebastian-peter** [comment]
This is blocked by https://github.com/ie3-institute/PowerSystemDataModel/pull/443

**Comment by ckittl** [comment]
Please also consider adapting the `pv_input.csv` files in the `input` directory.

**Comment by danielfeismann** [comment]
> Please also consider adapting the `pv_input.csv` files in the `input` directory.

done

File: src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala [hunk 1]
```diff
@@ -559,7 +559,7 @@ object WeatherSource {
     */
   object WeatherScheme extends ParsableEnumeration {
     val ICON: Value = Value("icon")
-    val PSDM: Value = Value("psdm")
+    val Cosmo: Value = Value("psdm")
```
Comment (sebastian-peter) [inline src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala:562]: Please rename to uppercase `COSMO` and also change the String parameter to `"cosmo"`

**Review by sebastian-peter — CHANGES_REQUESTED** [review]

File: src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala (hunk 1 above)
Comment (danielfeismann) [reply src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala]: Done

**Review by sebastian-peter — APPROVED** [review]
Nice work!

**Review by t-ober — CHANGES_REQUESTED** [review]
Looks good to me but in the `WeatherSourceWrapper` line 26 and `WeatherSourceWrapperSpec` line 11 the `WeatherSource` the WeatherSource gets renamed to `PsdmWeatherSource`. Shouldn't this be renamed too?

**Review by t-ober — APPROVED** [review]

**Pull request merged**